qiskit/tools/parallel.py

```diff
@@ -111,7 +111,14 @@ def _callback(_):
        and os.getenv('QISKIT_IN_PARALLEL') == 'FALSE':
         os.environ['QISKIT_IN_PARALLEL'] = 'TRUE'
         try:
-            pool = Pool(processes=num_processes)
+            if platform.system() == 'Darwin':
```

Comment:
Can we also add a check here for python version to only run this on python 3.8, something like:

```suggestion
            if platform.system() == 'Darwin' and sys.version_info[0] == 3 and sys.version_info[1] == 8:
```

I don't think we want to change the spawn method for python 3.5, 3.6, or 3.7 since everything seems to work fine on the older versions. This way we just limit the workaround to only the version it is needed

Reply:
I agree with you. Thanks!